configure.ac

```diff
@@ -1455,7 +1455,8 @@ opal_show_title "Setup PRRTE support"
 
 m4_ifdef([project_prrte],
          [OMPI_SETUP_PRRTE],
-         [ompi_want_prrte=no])
+         [OPAL_SUMMARY_ADD([[Miscellaneous]],[[PRRTE]],[prrte],[no (disabled)])
+          ompi_want_prrte=no])
```

Comment:
Should we also have a Miscellaneous / PRRTE line for showing whether PRRTE is embedded or external?

Reply:
We don't support external PRRTE, remember? You guys didn't see a need for it